src/iconsManifest/supportedExtensions.ts

```diff
@@ -1052,6 +1052,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'dojojs',
```

Comment:
I would name it simply `dojo`. https://dojo.io/

Reply:
I got lazy and updated the files at GitHub interface. Hopefully it is ok now :D